### DISCUSSION TITLE
Entropy - TESpy

### DISCUSSION THREAD

**Discussion by zain-oday** [oemof/tespy#467] [Q&A]
Hello everyone,
My question is: when I run any code to design a particular cycle by TESpy, the results column does not show up at the entropy , it just shows up  m,P,h and T.
I'm looking forward to the solution.

best regards

**Comment by fwitte** [comment]
Hi @zain-oday,

I was in holidays, so a little late...

`nwk.print_results` only prints mass flow, pressure, enthalpy and temperature. But you can access all results of a network via pandas DataFrames:

```python
nwk.results['Connection']
```

Maybe it would be worth it, that users can specify, which results to print, for example `nwk.print_results(connection_parameters=["m", "s", "h"])`?

You can also access the data via each connection object

```python
conn.s.val_SI
```

Information should be available in the docs, see: https://tespy.readthedocs.io/en/main/modules/networks.html#tespy-modules-networks-label

Best
